acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/Signer.java

```diff
@@ -152,6 +153,10 @@ public Response callApiPublicKeys(final KeyType keyType) {
     return given().baseUri(getUrl()).get(publicKeysPath(keyType));
   }
 
+  public List<String> listPublicKeys(final KeyType keyType) {
+    return given().baseUri(getUrl()).get(publicKeysPath(keyType)).as(new TypeRef<>() {});
```

Comment:
any reason this doesn't call into callApiPublicKeys?

Reply:
no I like that. done